cms/djangoapps/contentstore/views/block.py

```diff
@@ -301,20 +302,25 @@ def xblock_view_handler(request, usage_key_string, view_name):
         return HttpResponse(status=406)
 
 
+@xframe_options_exempt
 @require_http_methods("GET")
 @login_required
-def edit_view_xblock(request, usage_key_string):
+def xblock_actions_view(request, usage_key_string, action_name):
     """
     The handler for rendered edit xblock view.
```

Comment:
Please update Docstring to describe the rendering for different xblock actions

"""
Return rendered xBlock action view. 
The action name should be provided as an argument.
Valid options for action name are edit and move.
"""